requirements.txt

```diff
@@ -0,0 +1 @@
+matplotlib==2.2.5
```

Comment:
Cool!

Reply:
yep and you should keep using it, when you install a dep via pip, you have to add it here (sorry pip doesn't seem to do that automatically but pipenv does maybe that's worth looking into)